src/main/java/org/gaul/s3proxy/S3Proxy.java

```diff
@@ -145,6 +151,8 @@ public static final class Builder {
         private CrossOriginResourceSharing corsRules;
         private int jettyMaxThreads = 200;  // sourced from QueuedThreadPool()
         private int maximumTimeSkew = 15 * 60;
+        private static final Logger logger = LoggerFactory.getLogger(
```

Comment:
I think it should be OK to use the `S3Proxy` logger you added above.  But you might need to remove the `private` access modifier so it is package-protected.